css/style.css

```
@@ -0,0 +1,411 @@
+.wrapper {
+  position: relative;
+}
+.wrapper .slider-main {
+  position: absolute;
+  top: 0rem;
+  bottom: 0rem;
+  left: 0rem;
+  right: 0rem;
+  width: 100%;
+  height: 95.9rem;
+  z-index: 2;
+  background-color: #2c2e3d;
+  max-width: 192rem;
+  margin: 0 auto;
+  position: relative;
+  overflow: hidden;
+  margin-bottom: 20rem;
+}
+.wrapper .slider-main .slider__arrows .slider__arrow-left {
+  position: absolute;
+  top: calc(50% - 4rem);
+  bottom: unset;
+  left: unset;
+  right: unset;
+  width: 8rem;
+  height: 8rem;
+  z-index: 4;
+  --webkit-display: flex;
+  --moz-display: flex;
+  --ms-display: flex;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  cursor: pointer;
+  color: #ffffff;
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.05) 0px 4px 6px -2px;
+  background-color: #212331;
+}
+.wrapper .slider-main .slider__arrows .slider__arrow-left:hover {
+  background-color: #00bcd4;
+  cursor: pointer;
+}
+.wrapper .slider-main .slider__arrows .slider__arrow-left .arrow__icon {
+  font-size: 4.4444444444rem;
+}
+.wrapper .slider-main .slider__arrows .slider__arrow-right {
+  position: absolute;
+  top: calc(50% - 4rem);
+  bottom: unset;
+  left: unset;
+  right: unset;
+  width: 8rem;
+  height: 8rem;
+  z-index: 4;
+  --webkit-display: flex;
+  --moz-display: flex;
+  --ms-display: flex;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  cursor: pointer;
+  color: #ffffff;
+  box-shadow: rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.05) 0px 4px 6px -2px;
+  background-color: #212331;
```

những màu dùng nhiều như này em bỏ vào biến đặt trong :root ở base.css nhé
xem những màu nào dùng nhiều đặt biến chung luôn

---

1. Em có đặt bên Variable.scss file 1 số biến kiểu thế này
```suggestion
  $primary-color: #99a9b5;
  $secondary-color: #00bcd4;
  $white-color: #ffffff;
  $gray-color: #6c7c86;
  $black-color: #3c3950;

  $bg-dark-color: #393d50;
  $bg-light-color: #f0f3f2;
```

2. Những phần code khác em cũng có đặt chung thế này
```suggestion
  @mixin textStyle(
  $size: 2,
  $text-transform: lowercase,
  $color: white,
  $weight: normal
) {
  font-size: #{$size}rem;
  text-transform: $text-transform;
  color: $color;
  font-weight: $weight;
}
```